lib/platforms.nix

```diff
@@ -8,8 +8,9 @@ rec {
   openbsd = ["i686-openbsd" "x86_64-openbsd"];
   netbsd = ["i686-netbsd" "x86_64-netbsd"];
   cygwin = ["i686-cygwin" "x86_64-cygwin"];
-  unix = linux ++ darwin ++ freebsd ++ openbsd;
-  all = linux ++ darwin ++ cygwin ++ freebsd ++ openbsd;
+  illumos = ["x86_64-solaris"];
```

Comment:
It's a long story: https://www.youtube.com/watch?v=-zRN7XLCRhc

Basically Sun started Solaris, opensourced it (OpenSolaris) - got bought by Oracle which tried to close the source again, and was forked as illumos.

They're still binary compatible, but this it's hard to say what Oracle will do in the future.